js/plugins/vertexai/src/embedder.ts

```diff
@@ -39,6 +39,14 @@ export const VertexEmbeddingConfigSchema = z.object({
   title: z.string().optional(),
   location: z.string().optional(),
   version: z.string().optional(),
+  /**
+   * The `dimensions` parameter allows you to specify the dimensionality of the embedding output.
+   * By default, the model generates embeddings with 768 dimensions. Models such as
+   * `text-embedding-004`, `text-embedding-005`, and `text-multilingual-embedding-002`
+   * allow the output dimensionality to be adjusted between 1 and 768.
+   * By selecting a smaller output dimensionality, users can save memory and storage space, leading to more efficient computations.
+   **/
+  dimensions: z.number().min(1).max(768).optional(),
```

Comment:
same thing on the naming -- let's call it `outputDimensionality` for consistency